src/libraries/Microsoft.PowerFx.Core/Entities/External/TableDelegationInfo.cs

```diff
@@ -37,6 +38,9 @@ public abstract class TableDelegationInfo
         // Supports per record permission
         internal bool SupportsRecordPermission { get; init; }
 
+        [Obsolete("preview")]
+        internal bool SupportsJoinFunction { get; init; }
```

Comment:
>internal [](http://example.com/codeflow?start=8&length=8)

This is going to need to be public so that other shims can implement it.

Reply:
good point
I was too conservative